File: RPT2018E-master/TrabalhoED2PT1/src/Questions.cpp
#include "Questions.h"

Questions::Questions()
{
    //ctor
}

Questions::Questions(int id1, string user1,string data1,string score1,string title1)
{
    id = id1;
    user = user1;
    data = data1;
    score = score1;
    title = title1;
}
Questions::~Questions()
{
    //dtor
}

void Questions::imprime()
{
    cout<< id<<endl;
    cout<< user<<endl;
    cout<<title<<endl;
}

File: Trb/src/trb/Questions.java
package trb;

public class Questions {
    String id;
    String user,data,score,title;

    public Questions(String id, String user, String data, String score, String title) {
        this.id = id;
        this.user = user;
        this.data = data;
        this.score = score;
        this.title = title;
    }
    
   void imprime()
   {
        System.out.println("id "+id);
        System.out.println("usr " +user);
        System.out.println("data " +data);
        System.out.println("score "+score);
        System.out.println("title "+title);
   }
}
